### PR TITLE
Ensure `oc image info` works with manifest lists

### PR DESCRIPTION
**Pull request by smarterclayton**
Print an error that lists the available images and let the user select
one with --filter-by-os.

```
oc image info registry.access.redhat.com/rhel7:latest
error: the image is a manifest list and contains multiple images - use --filter-by-os to select from:

  OS            DIGEST
  linux/amd64   sha256:2fb67122fa3a8e14565016865ca67402e00f055ba7dba6febd8999b650beef72
  linux/ppc64le sha256:806a6140f248c8eca39b2f3a98469415c08c575e17e5d7a733e3d31df3df83ca
  linux/s390x   sha256:58e18319993c8f61dbb09c24167b638ae14d66d62dda92c585a40ce37b4964fc

```

### CODE REVIEW

File: pkg/oc/cli/image/info/info.go [hunk 1]
```diff
@@ -72,6 +74,9 @@ func (o *InfoOptions) Complete(cmd *cobra.Command, args []string) error {
 	if len(args) < 1 {
 		return fmt.Errorf("info expects at least one argument, an image pull spec")
 	}
+	if err := o.FilterOptions.Validate(); err != nil {
```
Comment (soltysh) [inline pkg/oc/cli/image/info/info.go]: `Complete` should fill in the necessary bits, not perform validation. Even if you're validating internal structure you should create a separate `Validate` method for `InfoOptions` and call `FilterOptions`' `Validate` there.

Comment (smarterclayton) [reply pkg/oc/cli/image/info/info.go]: I am not going to refactor this now.  We regressed and this gets us back to working for the use case at hand.  I'm happy to come back later.

Comment (smarterclayton) [reply pkg/oc/cli/image/info/info.go]: Later = post kubecon

**Pull request merged**